content/doc/book/pipeline/pipeline-best-practices.adoc

```diff
@@ -68,6 +68,16 @@ Using Jenkins.instance or its accessor methods in a Pipeline or shared library i
 
 Solution: The best solution would be to work around the calls being made, but if they must be done then it would be better to implement a Jenkins plugin which is able to gather the data needed.
 
+=== Cleaning up old Jenkins builds
+
+As a Jenkins administrator, removing old or unwanted builds keeps the Jenkins controller running efficiently.
+When you do not remove older builds, there are less resources for more current and relevant builds.
+This video reviews using the `build.Discarder` directive in a Pipeline for Global and Specific usage.
```

Comment:
Use the correct spelling of the declarative Pipeline directive.  Link to the more detailed documentation.
```suggestion
This video reviews using the link:/doc/book/pipeline/syntax/#options[`buildDiscarder`] directive in individual Pipeline jobs.
```